stix_shifter_modules/guardium/configuration/config.json

```diff
@@ -23,24 +42,36 @@
             "type": "password",
             "optional": true
         },
-        "client_id": {
-            "type": "text",
-            "previous": "connection.options.client_id"
-        },
-        "client_secret": {
-            "type": "password",
-            "previous": "connection.options.client_secret"
-        }
+        "options": {
+            "client_id": {
+                "type": "string",
+                "optional": true,
+                "hidden": true
+            },
+            "client_secret": {
+                "type": "password",
+                "optional": true,
+                "hidden": true
+            },
+            "grant_type": {
+                "type": "string",
+                "optional": true,
+                "hidden": true
+            }
+        } 
     },
     "configuration": {
         "auth": {
-            "type" : "fields",
```

Comment:
`"type" : "fields"` shouldn't be removed.

Reply:
I added it back.